packages/aws-lambda-otel-extension/opt/otel-extension/external/otel-payloads.js

```diff
@@ -169,14 +169,15 @@ const createLogPayload = (fun, logs) => {
   return logs.map((log) => {
     const split = (log.record || '').split('\t');
     return {
-      Timestamp: split[0] ? new Date(split[0]).getTime() : new Date().getTime(),
```

Comment:
It'll be good to confirm it's actually a date stamp, otherwise we may produce a `NaN` for any non `console.*` log that contains `\t` character

Reply:
@medikoo This is the old code that I am trying to replace with `Timestamp: new Date(log.time).getTime()`. Did you mean to comment on this part?

Reply:
ah.. sorry, not sure where I was looking at, ignore this comment :)

Reply:
Awesome, no worries man 😎